src/lightning_lite/strategies/parallel.py

```diff
@@ -0,0 +1,129 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from abc import ABC, abstractmethod
+from contextlib import contextmanager
+from typing import Any, Dict, Generator, List, Optional
+
+import torch
+from torch import Tensor
+from torch.nn import Module
+
+import lightning_lite as lite
+from lightning_lite.accelerators.accelerator import Accelerator
+from lightning_lite.plugins.environments.cluster_environment import ClusterEnvironment
+from lightning_lite.plugins.io.checkpoint_plugin import CheckpointIO
+from lightning_lite.plugins.precision import Precision
+from lightning_lite.strategies.strategy import Strategy
+from lightning_lite.utilities.distributed import (
+    _get_process_group_backend_from_env,
+    all_gather_ddp_if_available,
+    get_default_process_group_backend_for_device,
+    ReduceOp,
+)
+from lightning_lite.utilities.rank_zero import rank_zero_deprecation
+
+
+class ParallelStrategy(Strategy, ABC):
+    """Plugin for training with multiple processes in parallel."""
```

Comment:
```suggestion
    """Strategy for training with multiple processes in parallel."""
```

Reply:
After all this time, these old plugin references still pop up 😄